tests/s-longjmp.c

```diff
@@ -10,6 +10,7 @@ int foo(void)
 
 int bar(void)
 {
+	asm(".p2align 4");
```

Comment:
Could you please add a comment what it is and why it's needed?

Reply:
@namhyung Compiler Optimization level `>= -O1 ` make function `bar ()` no called( #1576 ).
```bash
===========  result  ===========
main() {
   _setjmp();
   foo() {
     longjmp() {
   } /* _setjmp */
 } /* main */
=========== expected ===========
main() {
   _setjmp();
   foo() {
     longjmp() {
   } /* _setjmp */
   bar();
 } /* main */
t015_longjmp: diff result of -pg -O1
--- expect      2022-09-12 11:25:44.222908251 +0900
+++ result      2022-09-12 11:25:44.222908251 +0900
@@ -5,3 +5,2 @@
    } /* _setjmp */
-   bar();
  } /* main */
```